packages/execution-environments/src/web-workers/WebWorkerController.ts

```diff
@@ -0,0 +1,24 @@
+import ObjectMultiplex from '@metamask/object-multiplex';
+import { WorkerPostMessageStream } from '@metamask/post-message-stream';
+import pump from 'pump';
+import { BaseController } from '../common/BaseControllers';
+import { SNAP_STREAM_NAMES } from '../common/enums';
+
+export class WebWorkerController extends BaseController {
```

Comment:
In line with the suggested renaming:

```suggestion
export class WebWorkerSnapExecutor extends BaseSnapExecutor {
```